plugin/hub/hub.go

```diff
@@ -57,240 +62,222 @@ I8Hs4e6+i4/g8yyp1aO9jClsLVJL4Xp9o6O6aYpSDj17MEXhV5U053grDEuvvNCA
 NdQkdLbveQ+US4vVAzRFJjRAvGVq14lRxiTreQ==
 =9Zuc
 -----END PGP PUBLIC KEY BLOCK-----`
-
-
-const (
-	defaultOrganization = "cloudquery"
-	defaultVersion = "latest"
 )
 
-func sha256File(filePath string) (string, error){
-	f, err := os.Open(filePath)
+type Hub struct {
+	registryURL     string
+	skipRegistry    bool
+	pluginDirectory string
+}
+
+func NewHub(skipRegistry bool) Hub {
+	dir, err := os.Getwd()
 	if err != nil {
-		return "", err
```

Comment:
why not return `nil`, err ?

Also same question as above - why not `*Hub` as return value?